en/sim_gazebo_gz/vehicles.md

```diff
@@ -41,6 +41,18 @@ make px4_sitl gz_x500_depth
 
 ![x500 with depth camera in Gazebo](../../assets/simulation/gazebo/vehicles/x500_depth.png)
 
+### X500 Quadrotor with Monocular Camera
+
+This models has a simple monocular camera sensor attached (there is no physical camera visualization on the model itself).
+
+```sh
+make px4_sitl gz_x500_mono_cam
+```
+
+:::note
+(18.12.2023) To enjoy the full functionality of the camera, a camera manager and ideally a GST plugin need to be provided. This is currently not available, but an [issue](https://github.com/PX4/PX4-Autopilot/issues/22563) has been opened to request these features.
```

Comment:
I think we need to be more direct. How about this? Note, they can follow the link to find out how old the issue is and what work needs to be done. Here I just want them to know that using this is not enough to enable the use cases you'd imagine it would enable.

```suggestion
The camera cannot yet be used to stream video or for image capture in QGroundControl.
[PX4-Autopilot#22563](https://github.com/PX4/PX4-Autopilot/issues/22563) can be used to track the additional work needed to fully enable these use cases.
```

Reply:
I'm assuming you are happy with this!